api/envoy/extensions/network/dns_resolver/getaddrinfo/v3/getaddrinfo_dns_resolver.proto

```diff
@@ -27,4 +29,7 @@ option (udpa.annotations.file_status).package_version_status = ACTIVE;
 //   Resolutions currently use a hard coded TTL of 60s because the getaddrinfo() API does not
 //   provide the actual TTL. Configuration for this can be added in the future if needed.
 message GetAddrInfoDnsResolverConfig {
+  // Specifies the number of retries the resolver will retry before giving up. If not specified,
+  // the resolver will retry retry indefinitely until it succeeds.
```

Comment:
nit: one extra "retry".  Also, maybe we can phrase this as:
Specifies the number of retries before the resolver gives up. If not specified, the resolver will retry indefinitely until it succeeds or the DNS query times out.

Also, maybe this should be the total number of tries, so we don't have to do the `+1` below

Reply:
I looked at the existing API and using `num_retries` is more common. That's why I went with `num_retries`: https://github.com/search?type=code&auto_enroll=true&q=repo%3Aenvoyproxy%2Fenvoy+num_retries+path%3Aproto